### ISSUE TITLE
[Feature Request] AFK detection/kick

### ISSUE THREAD

**Issue by lionkor** [BeamMP/BeamMP-Server#108]
**Is your feature request related to a problem? Please describe.**

Some servers which are very popular will have players in it which appear completely AFK.

**Describe the solution you'd like**

A config option to kick people after N minutes of inactivity, or maybe triggering an event like onAFKDetected or something.

**Describe alternatives you've considered**

There are already Lua plugins for this, but currently the server doesnt (yet) expose vehicle positions to the lua, so its always extra bandwidth for these types of plugins.

**Comment by Starystars67** [comment]
Hey Lion, Neat idea!

Just wondering if maybe it might be easier and make more sense to just make the vehicle positions available for server lua scripts?
This can then also be used for other purposes too down the road.

**Comment by lionkor** [comment]
Yes, thats a planned feature. I still think AFK kicking should be a core feature.
